plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoColumnHandle.java

```diff
@@ -114,11 +113,6 @@ public boolean equals(Object obj)
     @Override
     public String toString()
     {
-        return toStringHelper(this)
-                .add("name", name)
-                .add("type", type)
-                .add("hidden", hidden)
-                .add("comment", comment)
-                .toString();
+        return name + ":" + type;
```

Comment:
Updated `toString` method for `BaseConnectorTest#testPredicateReflectedInExplain`